components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -101,9 +104,15 @@ export const WorkspaceEntry: FunctionComponent<Props> = ({ info, shortVersion })
 };
 
 export function getProjectPath(ws: Workspace) {
-    if (CommitContext.is(ws.context)) {
-        return `${ws.context.repository.host}/${ws.context.repository.owner}/${ws.context.repository.name}`;
-    } else {
-        return undefined;
+    if (!ws.status?.gitStatus?.cloneUrl) {
+        return ws.contextUrl.replace("https://", "");
     }
```

Comment:
Are there concrete situations where the clone URL isn't present? Can we make the field more reliable across the API?

Also: is the output not going to have a `.git` suffix? Being conscious of some HTTP-only self-hosted SCMs, this would not strip the protocol there correctly as well. Maybe we can bring in a function like `stripOffProtocol`
https://github.com/gitpod-io/gitpod/blob/7af051548b26e8dfdfc3e372ce5856f357743b4a/components/dashboard/src/components/RepositoryFinder.tsx#L226-L231

Reply:
We only allow https SCMs, @iQQBot once want to have `http` support in his self-hosted Gitpod Instance, but feature request is rejected 😂 (If I remember it correctly)